content/en/docs/tasks/traffic-management/egress/egress-control/index.md

```diff
@@ -140,7 +140,18 @@ any other unintentional accesses.
 
 ### Access an external HTTP service
 
-1.  Create a `ServiceEntry` to allow access to an external HTTP service:
+1.  Create a `ServiceEntry` to allow access to an external HTTP service.
+
+    {{< warning >}}
+    The `DNS` resolution is used in the service entry below as a security measure. If the resolution were `NONE`,
```

Comment:
This took me a few times to read. This is indeed an important thing to call out. Do we have this security risk described somewhere else that we can just link to? If not, can I suggest a different wording:

The `DNS` resolution is used in the service entry below as a security measure. If the resolution was configured as `NONE`, this opens an attack vector. Malicious clients could claim to the Istio sidecar that they are trying to reach to `httpbin.org` (via putting that in the HOST header) but actually be connecting to a different IP (that is not associated with httpbin.org). The Istio sidecar proxy will trust the HOST header, and incorrectly allow the traffic, even though it is being delivered to the IP address of a different host.

By configuring resolution as `DNS`, the sidecar proxy... (rest was clear)

Reply:
@ssuchter Another take, please take a look at https://github.com/istio/istio.io/pull/6217/commits/ff5f7b5edcc46336611055676d9fe1581f1b7cbe.

Reply:
LGTM